plugins/modules/device_credential_intent.py

```diff
@@ -25,6 +25,10 @@
 author: Muthu Rakesh (@MUTHU-RAKESH-27)
         Madhan Sankaranarayanan (@madhansansel)
 options:
+  dnac_log_level:
+    description: Log levels are used to categorize the logs based on their severity.
+    type: str
+    default: INFO
```

Comment:
How the customers know other options?
Can we specify the choices?
choices: [ CRITICAL, ... INFO,... ]